.env

```diff
@@ -54,6 +54,7 @@ UBUNTU=22.04
 
 # Default versions for various dependencies
 CLANG_TOOLS=14
+CMAKE=3.25.0
```

Comment:
Would it be possible (and also would it be advisable...) to call this `CMAKE_VERSION` instead? One of the R failures I suspect might be due to us looking to `CMAKE` for a path to the cmake executable itself at https://github.com/apache/arrow/blob/bcb5bd30d0452a95ed7c44fc3bd6c8e3f67faab1/r/tools/nixlibs.R#L695 (though I don't have a full traceback I'm not 100% certain on that)

Reply:
Aaahi didn't see the previous comments on the PR (sorry, I must have had a tab open and didn't refresh!) but I suspect this here is the answer to:

> It would probably be more forward-looking to avoid the error entirely. Why does the function fail parsing the CMake version?